nfpm.go

```diff
@@ -48,6 +50,18 @@ func Parse(in io.Reader) (config Config, err error) {
 	config.Info.Release = os.ExpandEnv(config.Info.Release)
 	config.Info.Version = os.ExpandEnv(config.Info.Version)
 
+	// parse the version as a semver so we can properly split the parts and support proper ordering for both rpm and deb
```

Comment:
this entire block should be in the `withdefaults` method, as when using goreleaser as a lib this method is not used.

Reply:
Got it, sry